### ISSUE TITLE
Demo server not providing LMS sigGen vectors

### ISSUE THREAD

**Issue by jvdsn** [usnistgov/ACVP-Server#263]
**environment**
Demo

**testSessionId**
405967

**vsId**
1655897

**Algorithm registration**
```
      {
        "revision":"1.0",
        "algorithm":"LMS",
        "mode":"sigGen",
        "capabilities":{
          "lmsModes":[
            "LMS_SHA256_M24_H5",
            "LMS_SHA256_M24_H10"
          ],
          "lmOtsModes":[
            "LMOTS_SHA256_N24_W1",
            "LMOTS_SHA256_N24_W8"
          ]
        },
        "specificCapabilities":[
        ],
        "prereqVals":[
          {
            "algorithm":"SHA",
            "valValue":"same"
          },
          {
            "algorithm":"DRBG",
            "valValue":"same"
          }
        ]
      },
```

**Expected behavior**
Server provides test vectors.

**Additional context**
When sample vectors are requested, the server *does* provide test vectors. See testId 405966, vsId 1655894

**Comment by jvdsn** [comment]
Presumably related to #252

**Comment by livebe01** [comment]
Letting you know that we've received this and are looking into it.

**Comment by jvdsn** [comment]
I can confirm this is fixed on the demo server

**Comment by livebe01** [comment]
The fix for this is now on Demo in release v1.1.0.29.

**Comment by livebe01** [comment]
The fix for this is on Prod in release [v1.1.0.29-hotfix-1](https://github.com/usnistgov/ACVP-Server/releases/tag/v1.1.0.29-hotfix-1).
